pkg/action/hooks_test.go

```diff
@@ -264,6 +268,14 @@ func (h *HookFailingKubeClient) Delete(resources kube.ResourceList) (*kube.Resul
 	return h.PrintingKubeClient.Delete(resources)
 }
 
+func (h *HookFailingKubeClient) GetWaiter(strategy kube.WaitStrategy) (kube.Waiter, error) {
```

Comment:
Whithout all of this I had this error
```
--- FAIL: TestHooksCleanUp (0.00s)
    --- FAIL: TestHooksCleanUp/Deletion_hook_runs_for_previously_successful_hook_on_failure_of_a_heavier_weight_hook (0.00s)
        /Users/benoit.tigeot/projects/lifen/helm/pkg/action/hooks_test.go:374: Expected and error but did not get it.
```

The issue seems that HookFailingKubeClient doesn't had a GetWaiter method that returns a properly failing waiter.